dep-tasks.yaml

```
@@ -0,0 +1,39 @@
+
+  mariadb-connector-c:
+    internal: true
+    platforms: ["linux"]
+    dir: "{{.G_LIB_INSTALL_DIR}}"
+    cmds:
+      - "./mariadb-connector-c.sh {{.MARIADB_CONNECTOR_C_VERSION}}"
+
+  submodules:
+    internal: true
+    cmds:
+      - "git submodule update --init --recursive"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)